ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/EngineExchangeCapabilities.java

```diff
@@ -63,7 +63,6 @@ public JsonRpcResponse syncResponse(final JsonRpcRequestContext requestContext)
             .filter(e -> e.getMethodName().startsWith("engine_"))
             .filter(e -> !e.equals(ENGINE_EXCHANGE_CAPABILITIES))
             .filter(e -> !e.equals(ENGINE_PREPARE_PAYLOAD_DEBUG))
-            .filter(e -> !e.getMethodName().endsWith("6110"))
```

Comment:
Should pre-prague Besu advertise that it has engine_*V4 available yet 🤔 ?

Reply:
They will be filtered out at `ExecutionEngineJsonRpcMethods` if we are not in Prague.